src/ipyaladin/__init__.py

```diff
@@ -119,6 +130,27 @@ def _handle_custom_message(self, model, message, list_of_buffers):  # noqa: ARG0
         elif event_type == "select" and "select" in self.listener_callback:
             self.listener_callback["select"](message_content)
 
+    @property
+    def fov(self) -> Angle:
+        """The field of view of the Aladin Lite widget.
+
+        It can be set with either a float or an `~astropy.units.Angle` object.
```

Comment:
"The field of view of the Aladin Lite widget along the horizontal axis.

It can be set with either a float in degrees or an ..."